style.css

```diff
@@ -1,469 +1,490 @@
-  top: 4528px;
-  left: 49.5%;
-  transform: translate(-50%, -50%);
-}
-
-.num2 {
-  position: absolute;
-  top: 4909px;
-  left: 49.5%;
-  transform: translate(-50%, -50%);
-}
-
-.num3 {
-  position: absolute;
-  top: 5294px;
-  left: 49.5%;
-  transform: translate(-50%, -50%);
-}
-
-.frame-regist {
-  display: flex;
-  justify-content: flex-end;
-}
-
-.frame4 {
-  width: 327px;
-  position: absolute;
-  top: 5688px;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  color: #172b4d;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  text-align: center;
-}
-
-form {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  width: 327px;
-  position: absolute;
-  top: 6100px;
-  left: 50%;
-  transform: translate(-50%, -50%);
-}
-
-.email {
-  text-decoration: none;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  width: 327px;
-  position: absolute;
-  top: 6280px;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  color: #42526e;
-}
-
-label {
-  font-weight: bold;
-  margin-bottom: 10px;
-}
-
-input,
-textarea {
-  padding: 8px;
-  border-radius: 4px;
-  border: 1px solid #ccc;
-  margin-bottom: 20px;
-  width: 100%;
-}
-
-.send {
-  background-color: #36b37f;
-  color: #fff;
-  border-radius: 4px;
-  border: none;
-  padding: 10px 20px;
-  cursor: pointer;
-  transition: background-color 0.2s;
-}
-
-.send:hover {
-  background-color: #2d8e69;
-}
-
-.frame-social {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  width: 327px;
-  position: absolute;
-  top: 6385px;
-  left: 50%;
-  transform: translate(-50%, -50%);
-}
-
-.social-media2 {
-  list-style: none;
-  display: flex;
-  flex-direction: row;
-  gap: 20px;
-  margin-top: 20px;
-  padding: 0 1rem;
-  align-items: center;
-  justify-content: center;
-}
-
-.stack {
-  display: flex;
-  justify-content: space-between;
-  flex-direction: row;
-  gap: 24px;
-  padding-top: 20px;
-  font-size: 12px;
-  align-items: flex-start;
-}
-
-.items {
-  color: #36b37e;
-  padding: 8px 12px;
-  background-color: white;
-  border-radius: 2px;
-}
-
-.figure1 {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  position: relative;
-  top: 3320px;
-  left: 150px;
-  margin-left: auto;
-  margin-right: auto;
-}
-
-.figure2 {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  position: absolute;
-  top: 4270px;
-  margin-right: auto;
-  margin-left: auto;
-}
-
-.figure3 {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  position: relative;
-  top: 5465px;
-  margin-right: 0.1px;
-  margin-left: auto;
-}
-
-@font-face {
-  font-family: 'Inter-for-Microverse';
-  src: url('inter.ttf') format('truetype');
-}
-
-.form-container ul {
-  list-style: none;
-  padding-left: 0;
-  width: 100%;
-}
-
-.form-container input {
-  font-size: 20px;
-}
-
-textarea {
-  height: 80px;
-  font-size: 20px;
+@media only screen and (min-width: 768px) and (max-width: 2442px) {
```

Comment:
- [Optional ] Great Job for Adding the media query, however, the main usage of the media query is to have a different viewing experience, so please apply the mobile style which you have done in the previous milestone 👌